### PR TITLE
How to use Remix IDE

### PR DESCRIPTION
**Pull request by AnmolSirola**
## Description

Made a complete guide of using Remix Ide

Fixes <#68>

## Type of change

- [x] Add a new project

- [x] Improve the readme of any project

## Domain of Contribution

- [x] Project using only Solidity

- [x] Project Test on the Remix IDE

## Checklist ✓

- [x] I'm SSOC'22 contributor

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
write like this as this is not linking with that issue   .
Fixes #68

**Comment by AnmolSirola** [comment]
Done Sir

**Comment by Vikash-8090-Yadav** [comment]
Ojk done one last thing attach the video at the last as this is blog so video is must , Sorry i forget to mention earlier , 
All set to merge . @AnmolSirola

**Comment by AnmolSirola** [comment]
@Vikash-8090-Yadav  Can u check it now ?

File: Blog/How to use remix IDE/How to use remix.ide.md [hunk 1]
```diff
@@ -39,3 +39,8 @@ We can see the ouput below the function name.
 
 ![output main](https://user-images.githubusercontent.com/95535448/188327607-21c3b977-0cf1-4d41-81b4-503fc385f4b4.png)
  
+### Working of a smart contract
+
+
+https://user-images.githubusercontent.com/95535448/188427582-a9540a5e-510e-47a7-9153-6b5e054dabcf.mp4
+
```
Comment (Vikash-8090-Yadav) [inline Blog/How to use remix IDE/How to use remix.ide.md:46]: Remove the sound coming from this video ..its irritating..🥲

Comment (AnmolSirola) [reply Blog/How to use remix IDE/How to use remix.ide.md:50]: I dont know how the sound is being produced -_- . I have made a new video plz check

**Comment by Vikash-8090-Yadav** [comment]
GREAT @AnmolSirola

**Pull request merged**